whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -1826,6 +1832,22 @@ class PostgreSQLComponent {
         }
     }
 
+    boolean iriIsLinkable(String iri, Connection connection) {
+        PreparedStatement preparedStatement = null
+        ResultSet rs = null
+        try {
+            preparedStatement = connection.prepareStatement(GET_IRI_IS_LINKABLE)
```

Comment:
connection is null here if called from `void normalizeIdentifiers(Document document)` ?